_posts/2023-05-10-omero-web-5.20.0.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: Release of OMERO.web 5.20.0
+intro-blurb: The OME team is pleased to announce the release of OMERO.web 5.20.0
+---
+
+Today we are releasing OMERO.web 5.20.0, which includes a fix for multi-file upload validation, 
+support for the [Django 3.2.19 security update]({{ site.baseurl }}/security/advisories/2023-31047/index.html), 
+and a number of bugfixes.
+
+Please see the [OMERO.web changelog](https://github.com/ome/omero-web/blob/v5.20.0/CHANGELOG.md).
+
+### Installing the Software
+
+OMERO.web 5.20.0 is available from PyPI - see 
+[here](https://pypi.org/project/omero-web/5.20.0/).
+
+Official Docker images are available as usual on Docker Hub with either
+the latest or the 5.20 tag:
+
+* <https://hub.docker.com/r/openmicroscopy/omero-web-standalone>
+* <https://hub.docker.com/r/openmicroscopy/omero-server>
```

Comment:
Should this be https://hub.docker.com/r/openmicroscopy/omero-web ?

Reply:
it should